### ISSUE TITLE
one test fails on main

### ISSUE THREAD

**Issue by behrica** [scicloj/noj#30]
There was a  numeric "chagne" in the prediction result introduced by latest commit:

```clojure
(deftest
 test57
 (is
  (=
   var56
   [["no" 0.6470588235294118 0.35294117647058826]
    ["no" 0.5714285714285714 0.42857142857142855]
    ["no" 0.8529411764705882 0.14705882352941177]

    ["no" 0.8879310344827587 0.11206896551724138] ;; working
    ["no" 0.8879310344827587 0.11206896551724138] ;; working
    ;["no" 0.9064327485380117 0.0935672514619883] ;; current code failing
    ;["no" 0.9064327485380117 0.0935672514619883] ;; current code failing
    ])))

```

My changes in branch #29 introduced this test and it was "once" working I believe.
Its a random forrest model and it might not give always same results.

I might use "approximative" comparison or see if I can fix the random seed.

**Comment by behrica** [comment]
fixed by  a40614d8a3a3b103a578a949d5c64cd72d91887e

**Comment by daslu** [comment]
Great!
